source/module_cell/parallel_kpoints.h

```diff
@@ -37,24 +37,35 @@ class Parallel_Kpoints
 #endif
 
     // information about pool, dim: GlobalV::KPAR
-    int* nproc_pool = nullptr;
-    int* startpro_pool = nullptr;
+    //int* nproc_pool = nullptr;    it is not used
+    //int* startpro_pool = nullptr;
+    std::vector<int> startpro_pool;
```

Comment:
The new variables in Class should not be public members, please refactor to use `set_xx()` and `get_xx()` interfaces to access them?